src/diffusers/__init__.py

```diff
@@ -16,6 +17,13 @@
 from .utils import logging
 
 
+# This will create an extra dummy file "dummy_torch_and_accelerate_objects.py"
+# TODO: (patil-suraj, anton-l) maybe import everything under is_torch_and_accelerate_available
+if is_torch_available() and not is_accelerate_available():
+    error_msg = "Please install the `accelerate` library to use Diffusers with PyTorch. You can do so by running `pip install diffusers[torch]`. Or if torch is already installed, you can run `pip install accelerate`."  # noqa: E501
+    raise ImportError(error_msg)
```

Comment:
the `fix-copies` is creating another file because of this `dummy_torch_and_accelerate_objects.py`

Will let it be for now, and adapt this in future PR, and probably add `is_torch_and_accelerate_available` and import everything under it cc @patrickvonplaten @anton-l @pcuenca